.github/workflows/deploy.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy main Branch
+
+on:
+  push:
+    branches:
+      - master
+      - new-deployment
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: GuillaumeFalourd/clone-github-repo-action@v2
+      with:
+        owner: 'deltachat'
+        repository: 'cosmos'
+    - name: Upload
+      uses: horochx/deploy-via-scp@1.1.0
```

Comment:
It's not auto-updating, it's pinned at version 1.1.0 - I don't know horochx though.

Reply:
note that we have been using this github action for years, for https://delta.chat.

Reply:
No, it also saves the private key to a file so it can be used by the scp command. Yes, I could write this in bash myself. But I could also just point to this github action where someone already did it.